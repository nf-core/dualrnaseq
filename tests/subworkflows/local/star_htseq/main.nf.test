nextflow_workflow {

    name "Test Workflow STAR_HTSEQ"
    script "subworkflows/local/star_htseq/main.nf"
    workflow "STAR_HTSEQ"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            workflow {
                """
                // define inputs of the workflow here. Example:
                input[0] = Channel.from('hello','nf-test')
                input[1] = Channel.from('hello','nf-test')
                input[2] = Channel.from('hello','nf-test')
                input[3] = Channel.from('hello','nf-test')
                input[4] = Channel.from('hello','nf-test')
                input[5] = Channel.from('hello','nf-test')
                """
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }

}
