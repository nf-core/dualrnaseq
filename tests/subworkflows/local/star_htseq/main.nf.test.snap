{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "timestamp": "2023-12-10T20:51:59.09991"
    }
}